Repositories/TimeTable.php

```diff
@@ -324,4 +323,27 @@ private function getAllProjectIds(): array
 
         return $projectIds;
     }
+
+    /**
+     * getAllUsers - Retrieves a list of all users from the database
+     *
+     * @access public
+     * @return array<string, mixed> An array containing user details, including ID, full name, and role
+     */
+    public function getAllUsers(): array
+    {
+        $sql = 'SELECT * FROM zp_user';
```

Comment:
is there a `where` clause you can add?

Reply:
I added a where clause for filtering out api users. I furthermore added a where status = 'a' which only hits active users.